binary_search_tree/binary_search_tree.py

```diff
@@ -5,10 +5,35 @@ def __init__(self, value):
     self.right = None
 
   def insert(self, value):
-    pass
+    if value > self.value:
+      if self.right is None:
+        self.right = [value]
+      else:
+        self.right.append(value)
+    elif value < self.value:
+      if self.left is None:
+        self.left = [value]
+      else:
+        self.left.append(value)
```

Comment:
It's not totally evident by the class, but the `BinarySearchTree` class actually spits out tree node objects, each with a `value`, `left`, and `right` attribute. The idea is that to create a tree structure, you'll assign tree node objects to be `left` and `right` objects, similar to how we assigned a node object to be the value of `next_node` of another node object to create a linkage.